eland/query_compiler.py

```diff
@@ -273,30 +283,30 @@ def _es_results_to_pandas(self, results, batch_size=None, show_progress=False):
             df = df[self.columns]
 
         if show_progress:
-            print("{}: read {} rows".format(datetime.now(), i))
+            print(f"{datetime.now()}: read {i} rows")
 
         return partial_result, df
 
     def _flatten_dict(self, y, field_mapping_cache):
-        out = OrderedDict()
+        out = {}
```

Comment:
We don't need to use `OrderedDict` anymore because all Python dicts are ordered, free performance boost. :)